src/deploy-events.js

```diff
@@ -9,10 +9,11 @@ module.exports = (client) => {
 
   for (const file of eventFiles) {
     const event = require(path.join(eventsPath, file));
-    if (event.once) {
-      client.once(event.name, event.execute);
-    } else {
-      client.on(event.name, event.execute);
-    }
+
+    const eventHandler = (...args) => event.execute(...args, client);
```

Comment:
This way we don't know in what position will be our `client` param, since discord could send 1 or 2 or 3 params and that would move our `client` param to the end.

Here I made an example where when Discord send 1 parameter then it works fine,but when sends two or more then the `client` param that we are receiving is not what I expected:

![image](https://github.com/user-attachments/assets/9ba54c1c-2d29-4bf7-a532-0c97643b659a)

To solve this I'd suggest move client as the first param and then the args as second param, so if discord send multiple params it would look like this:

`(client, discordParam1, discordParam2, discordParam3)`